plugins/plugin-urlfactory/src/main/java/org/eclipse/che/plugin/urlfactory/URLFactoryBuilder.java

```diff
@@ -40,17 +44,21 @@
   protected static final String DEFAULT_DOCKER_IMAGE = "codenvy/ubuntu_jdk8";
 
   /** Default docker type (if repository has no dockerfile) */
-  protected static final String MEMORY_LIMIT_BYTES = Long.toString(2000L * 1024L * 1024L);
+  protected static final String DEFAULT_MEMORY_LIMIT_BYTES = Long.toString(2000L * 1024L * 1024L);
 
   protected static final String MACHINE_NAME = "ws-machine";
 
-  /** Check if URL is existing or not */
-  @Inject private URLChecker URLChecker;
-
-  /** Grab content of URLs */
-  @Inject private URLFetcher URLFetcher;
+  private final URLChecker urlChecker;
+  private final URLFetcher urlFetcher;
+  private final FactoryMessageBodyAdapter factoryAdapter;
 
-  @Inject private FactoryMessageBodyAdapter factoryAdapter;
+  @Inject
```

Comment:
is there a reason to move to constructor based injection ?

Reply:
I like this way. At least because IDE doesn't show warnings that fields are not assigned